manifests/p/Panic/Playdate/1.1.0/Panic.Playdate.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Panic.Playdate

```

```suggestion
PackageIdentifier: Panic.Mirror
```
Since the EXE is `Mirror.exe`, the `PackageName` is `Mirror`, and there is no reference to a `Playdate`, should the `PackageIdentifier` be updated?

- Please remember to update this property in every file - and to update each file's path and name to match.